src/Client/Weather/Tomorrow/DataTransformer/TomorrowToWeatherDataTransformer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Client\Weather\Tomorrow\DataTransformer;
+
+use App\Model\Coordinates;
+use App\Model\WeatherData;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class TomorrowToWeatherDataTransformer
+{
+    private static array $weatherCode = [
+        "0" => 'Unknown',
+        '1000' => "Clear, Sunny",
+        '1100' => 'Mostly Clear',
+        '1101' => 'Partly Cloudy',
+        '1102' => 'Mostly Cloudy',
+        '1001' => 'Cloudy',
+        '2000' => 'Fog',
+        '2100' => 'Light Fog',
+        '4000' => 'Drizzle',
+        '4001' => 'Rain',
+        '4200' => 'Light Rain',
+        '4201' => 'Heavy Rain',
+        '5000' => 'Snow',
+        '5001' => 'Flurries',
+        '5100' => 'Light Snow',
+        '5101' => 'Heavy Snow',
+        '6000' => 'Freezing Drizzle',
+        '6001' => 'Freezing Rain',
+        '6200' => 'Light Freezing Rain',
+        '6201' => 'Heavy Freezing Rain',
+        '7000' => 'Ice Pellets',
+        '7101' => 'Heavy Ice Pellets',
+        '7102' => 'Light Ice Pellets',
+        '8000' => 'Thunderstorm',
+];
+
+private function roundToInt(float|int|null $value): ?int
+    {
+        return $value !== null ? (int)round($value) : null;
+    }
+
+    private function getWeatherDescription(int $code): string
+    {
+        if (self::$weatherCode[(string)$code])
+            return self::$weatherCode[$code];
+        return (string)$code;
+    }
+    public function transform(ResponseInterface $response): WeatherData
+    {
+        $data = $response->toArray();
+
+        return new WeatherData(
+            name: $data['location']['name'] ?? "",
+            coordinates: new Coordinates((string)$data['location']['lat'], (string)$data['location']['lon']),
+            weather: $this->getWeatherDescription($data['data']['values']['weatherCode']),
+            description: "",
```

Comment:
czym się rózni weather od description? Może to powinno być jedno pole? Troche mylące jest że w weatehr masz getWeatherDescription a w description nie masz nic. Btw nic to powinno być null zamiast empty string

Reply:
W OpenWeatherMap API wygląda to tak, że jest głowny określenie pogody "main", oraz opis
<img width="336" alt="image" src="https://user-images.githubusercontent.com/101008688/220065151-f0d54a47-ffc9-4fce-a5fc-b1561d5bd5b7.png">

Natomiast w Tomorrow nie ma opisu jest tylko kod, który ma przypisany do siebie jakoś opis
<img width="415" alt="image" src="https://user-images.githubusercontent.com/101008688/220065715-40ee0dc5-cef0-4961-b03a-5b61a9f31b4c.png">

Reply:
Zmienić nazwę funkcji na weatherCodeTransformer?